webapp/packages/core-ui/src/Form/FormState.ts

```diff
@@ -118,6 +118,10 @@ export class FormState<TState> implements IFormState<TState> {
     return this.dataContext.get(DATA_CONTEXT_LOADABLE_STATE).loaders.some(loader => loader.isCancelled?.() === true);
   }
 
+  isChanged(): boolean {
+    return Array.from(this.parts.values()).some(part => part.isChanged());
```

Comment:
it's not correct: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some#description

Reply:
I guess .some is okay here as we dont need to return an object and .some return true if at least one element in the array passes the test implemented by the provided function so it will not iterate the whole array